styles.css

```
@@ -18,11 +20,22 @@ a {
 
 main {
 	background: rgb(15, 45, 100);
+	height: 1000px;
+}
```

In mobile view, the button to generate a character is hidden behind the footer and inaccessible. Setting this line to use `min-height` would fix this.

```suggestion
	min-height: 1000px;
```